docs/RELEASE-NOTES.rst

```diff
@@ -4,6 +4,25 @@ Release Notes for BIG-IP Controller for Kubernetes
 |release|
 ---------
 
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch a subset of nodes with a user-specified label.
+* Creation of BIG-IP SSL Profiles from Kubernetes Secrets via Ingress TLS.
+* Creation of BIG-IP objects from OpenShift Route resources.
+  - This includes unsecured, Edge, Passthrough, and Re-encrypt Routes
+
+* This is a feature complete upgrade from the OpenShift F5Router. See the documentation for details
+  on how to upgrade: http://clouddocs.f5.com/containers/v1/openshift/replace-f5-router.html
+
+Bug Fixes
+`````````
+* Can properly configure http redirect rules on v11.6.1 BIG-IP systems
```

Comment:
I am no grammar/voice expert but I would drop 'Can' and add a period at the end to keep it consistent with other bullet items.